src/models.py

```diff
@@ -88,7 +88,29 @@ def build_model(args, vocab, pretrained_embs, tasks):
     rnn_params = Params({'input_size': d_emb, 'bidirectional': True,
                          'hidden_size': args.d_hid, 'num_layers': args.n_layers_enc})
 
-    if any(isinstance(task, LanguageModelingTask) for task in tasks) or \
+    if args.sent_enc=="onlstm":
```

Comment:
Make sure this is all in pep8 style. One easy option is to install autopep8 and run something like this:

`python -m autopep8 --max-line-length 100  --in-place --aggressive  --aggressive  ./*.py ./*/*.py ./*/*/*.py`